recipes/go/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "go" %}
+{% set version = "1.9.1" %}
+{% set sha256 = "a84afc9dc7d64fe0fa84d4d735e2ece23831a22117b50dafc75c1484f1cb550e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://storage.googleapis.com/golang/go{{ version }}.src.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Skip-unshare-and-capset-tests-on-Linux.patch  # [linux]
+    - 0001-Fix-for-TestRespectSetgidDir.patch  # [osx]
+
+build:
+  number: 0
+  # It's relocatable. $GOROOT FTW!
+  binary_relocation: False
+  # Worry about windows once the recipe is finalized
+  skip: True  # [win]
+
+requirements:
+  build:
+    - curl
+    - perl
+    - toolchain
+  # Optional (conda 4.4, where are you?)
+  # run:
+  #   - git
+
+test:
+  files:
+    - hello.go
+  commands:
+    - go version
+    - go help
+    - go run hello.go
+
+about:
+  home: http://golang.org
+  license: BSD-Style
```

Comment:
It looks like BSD 3-Clause. What motivates the "-Style"?

Reply:
The golang folks refer to the license as 'BSD-Style' everywhere in their docs. Although GH says that it is BSD 3-Clause.